### ISSUE TITLE
PNI: Test and Gather Feedback for Homepage Directions

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#5313]
Based on explorations in https://github.com/mozilla/foundation.mozilla.org/issues/5153#issuecomment-699223683

Test and gather feedback from audience to understand how they look for products to help decide which direction to pursuit. Compare two options to A/B test:

Homepage A: https://share.goabstract.com/8b35dce7-f8f8-4a7e-ba8c-fa2417911643
Homepage B: https://share.goabstract.com/dbe3aa68-17d2-4064-a9c9-eeb628ceb0b4
Presentation: https://docs.google.com/presentation/d/1-iaPgIvrXPv-4Qbr9CaAfmoqpJ724vubjJ--EikGV1E/edit?usp=sharing

**Comment by sabrinang** [comment]
I've iterated on the explorations of homepages to include ideas from both A and B to create homepage C: https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb

We no longer have two distinct directions to test. We also got feedback from Jen last week and feedback from the bi-weekly working group. I've also sent out async messages on slack channel to collect feedback as we will finalize the feedback at tomorrows design/dev prep call.

**Comment by sabrinang** [comment]
During our developer review, there was feedback with issues we were encountering with 'creepy'. I need to figure out the shared horizontal space with 'creepy' and on mobile the sticky elements were taking up too many vertical space. 

I'll explore solutions around making freeing 'creepy' from the top to be independent and we should be aware we should still clarify for users what 'creepy' does.

**Comment by sabrinang** [comment]
I’m refining PNI iteration to explore where to place the creepy face. Please review iterations in abstract (see relocate creepy / mobile sections): https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?sectionId=6ab91b1c-2eac-46be-b7fc-4776cba91f84

On desktop, I mocked up what it looks like if creepy started in the middle or if it only started on the bottom right. Another idea is that it relocated when you hit search or should we have search exposed if creepy is on the bottom right to begin with?

On mobile, creepy introduces itself in the middle to start, once you scroll past it then appears on the bottom right.

@kristinashu for design review

**Comment by sabrinang** [comment]
Last week I got design feedback on 'creepy' details (colour, size, location, shadow) and went with a tint of our blue close to the same size to what is currently live. 

I also checked in with Jen and overall she was fine with the styling direction but noted that she prefer the location of the face to be in an more obvious spot (upper right) however on mobile it was obscuring the browsing experience a bit too much as well as desktop so I've recommended lower right for both desktop/mobile but mentioned the positioning could be tested and adjusted if it gets lost (but ensured her since it is sticky and coloured it shouldn't be overlooked).

This direction has already been updated via abstract.
